.github/workflows/overcloud-host-image-build.yml

```diff
@@ -94,11 +94,15 @@ jobs:
         if: inputs.centos
 
       - name: Upload CentOS Stream 8 overcloud host image artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: CentOS Stream 8 overcloud host image
-          path: /opt/kayobe/images/overcloud-centos-stream-8
-          retention-days: 7
+        run: |
+          kayobe playbook run \
+          src/kayobe-config/etc/kayobe/ansible/upload-image.yml \
+          -i localhost \
+          -e image_path='/opt/kayobe/images/overcloud-centos-stream-8' \
+          -e os_distribution='centos' \
+          -e os_version='stream8'
+        env:
+          KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
```

Comment:
Ideally there would be some way to find out the URL of any images created. Perhaps as an artifact.

Reply:
Untested, but I'm adding a txt file artifact which will contain the URLs of all the new images

Reply:
tested, done